app/views/list/index.html.slim

```diff
@@ -0,0 +1,30 @@
+.container-fluid
+  .row
+    .col-xs-12
+      .wishlist-header
+        | Wishlists
+      .wishlist-page
+        - if @lists.present?
```

Comment:
We should display a message e.g. "You don't have any wishlists" if the variable is not present.

As an aside, `any?` works better than `present?` here, since invalid values like `1.present?` return true, but will blow up the rest of the page.